infrastructure/scripts/test_gpt5_api_access.py

```
@@ -0,0 +1,309 @@
+        print("\n2️⃣  Initializing GPT-5 Migration Client...")
+        try:
+            config = GPT5MigrationConfig(
+                primary_model=GPTModel.GPT_5,
+                fallback_model=GPTModel.GPT_4_TURBO,
+                enable_auto_fallback=True,
+                enable_prompt_optimization=False,  # Disable for testing
+            )
+
+            client = GPT5MigrationClient(api_key=os.getenv("OPENAI_API_KEY"), config=config)
+            print("✅ Client initialized successfully")
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)